data/test-cases/plugins/ms-teams-sync/mentions/Mention_multiple_users.md

```diff
@@ -0,0 +1,48 @@
+---
+# (Required) Ensure all values are filled up
+name: "Mention multiple users in channel or GM"
+status: Active
+priority: Normal
+folder: mentions
+authors: "@arush-vashishtha"
+team_ownership: []
+priority_p1_to_p4: P2 - Core Functions (Do core functions work?)
+
+# (Optional)
+location: null
+component: null
+tags: []
+labels: []
+tested_by_contributor: null
+
+# (Optional) Test type and tools
+cypress: null
+detox: null
+mmctl: null
+playwright: null
+rainforest: []
+manual_test_environments: []
+
+# Do not change
+id: null
+key: null
+created_on: null
+last_updated: null
+case_hashed: null
+steps_hashed: null
+---
+
+**Step 1**
+
+1. Send mention for multiple users in the linked channel or GM on MM.
```

Comment:
Make the steps consistent as mentioned above. Refer to the suggestion provided in the previous test case. Fix everywhere.